syncopy/specest/freqanalysis.py

```diff
@@ -291,20 +314,34 @@ def freqanalysis(data, method='mtmfft', output='pow',
 
     # Get everything of interest in local namespace
     defaults = get_defaults(freqanalysis)
+
     lcls = locals()
     # check for ineffective additional kwargs
     check_passed_kwargs(lcls, defaults, frontend_name="freqanalysis")
 
     new_cfg = get_frontend_cfg(defaults, lcls, kwargs)
 
+    is_fooof = False
+    if method == "mtmfft" and output.startswith("fooof"):
+        is_fooof = True
+        output_fooof = output
+        output = "pow"  # We need to change this as the mtmfft running first will complain otherwise.
+
+    if is_fooof:
+        fooof_output_types = list(fooofDTypes)
+        if output_fooof not in fooof_output_types:
+            lgl = "'" + "or '".join(opt + "' " for opt in fooof_output_types)
+            raise SPYValueError(legal=lgl, varname="output_fooof", actual=output_fooof)
+
     # Ensure a valid computational method was selected
     if method not in availableMethods:
         lgl = "'" + "or '".join(opt + "' " for opt in availableMethods)
         raise SPYValueError(legal=lgl, varname="method", actual=method)
 
     # Ensure a valid output format was selected
-    if output not in spectralConversions.keys():
-        lgl = "'" + "or '".join(opt + "' " for opt in spectralConversions.keys())
+    valid_outputs = list(spectralConversions)
```

Comment:
why not iterate over the keys directly as it was done before?

Reply:
No particular reason in the remaining code. I had temporarily created a `valid_outputs` list that was the combination of the `spectralConversions.keys()` and the valid fooof outputs, therefor I had to change this. now there is no reason anymore.

reverted in eb33667